### PR TITLE
refactor: Toast

### PR DESCRIPTION
**Pull request by Nerixyz**
# Description

* Fixes a bug where avatars weren't loaded on fresh installations of Chatterino.
* Avatars now update every two weeks.
* Removes misleading `DownlaodManager` (now part of `Toasts.cpp`).
* Refactors usage of WinToast to be easier to read.
* Added version to AUMI.
* Removes manual `QString` → `std::wstring` conversions.
* Removes uses of implicit ASCII casts in `Toasts.cpp`, meaning it can be compiled with `QT_NO_CAST_FROM_ASCII`.

### CODE REVIEW

File: src/singletons/Toasts.cpp [hunk 1]
```diff
@@ -49,24 +97,18 @@ bool Toasts::isEnabled()
...
-    auto iterator = Toasts::reactionToString.find(reaction);
-    if (iterator != Toasts::reactionToString.end())
-    {
-        return iterator->second;
-    }
-    else
-    {
-        return DONT_OPEN;
-    }
+    return reactionToString(reaction);
```
Comment (pajlada) [inline src/singletons/Toasts.cpp:100]: Just inline the `reactionToString` function in here

File: src/singletons/Toasts.cpp [hunk 2]
```diff
@@ -131,95 +172,143 @@ class CustomHandler : public WinToastLib::IWinToastHandler
...
+        if (this->file_.isOpen())
+        {
+            this->file_.close();
+        }
+        emit downloadComplete();
+        this->deleteLater();
+    });
+}
+
+#include "Toasts.moc"
```
Comment (pajlada) [inline src/singletons/Toasts.cpp:312]: What is this?

**Review by pajlada — CHANGES_REQUESTED** [review]

File: src/singletons/Toasts.cpp (hunk 2 above)
Comment (Nerixyz) [reply src/singletons/Toasts.cpp:308]: This is from the `AvatarDownloader : QObject`.

Comment (pajlada) [reply src/singletons/Toasts.cpp:308]: yuck but I assume this is because it's a class QObject in a source file

**Review by pajlada — APPROVED** [review]

**Pull request merged**